Resources/Views/Hub/home.leaf

```diff
@@ -0,0 +1,330 @@
+                <div class="modal-footer">
+                  <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close Modal (Keep Session)</button>
+                  <a href="/ticket/#(session.id)/cancel/#(slot.id)" class="btn btn-danger">Cancel</a>
+                </div>
+              </div>
+            </div>
+          </div>
+          #else:
+          <!-- Modal -->
+          <div class="modal fade" id="slot-#(slot.id)" tabindex="-1" aria-labelledby="slot-#(slot.id)-title" aria-hidden="true">
+            <div class="modal-dialog">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h1 class="modal-title fs-5" id="slot-#(slot.id)-title">Book Drop-in Session</h1>
+                  <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body">
+                  This will reserve a 15 minute session with #(session.owner) at #date(slot.date, "HH:mm") on #date(slot.date, "dd/MM/yyyy").
+                </div>
+                <div class="modal-footer">
+                  <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close Modal</button>
+                  <a href="/ticket/#(session.id)/book/#(slot.id)" class="btn btn-primary">Book</a>
+                </div>
+              </div>
+            </div>
+          </div>
+          #endif
+                
+                
+              #endfor
+              
+              </div>
+              #endfor
+              
+              <hr class="mt-3" />
+              #endif
+              
+              #if(count(groupSessions) > 0):
+                            
+              <!-- Focus Groups -->
+              <h2 id="focus" class="h5 text-primary pt-1 pt-lg-3 mt-4">Focus Groups</h2>
+              
+              <hr />
+              
+              #endif
+              
+              <!-- VISA Invitation -->
+              <h2 id="visa" class="h5 text-primary pt-1 pt-lg-3 mt-4">VISA Invitation</h2>
+              <p>We want to give everybody the opportunity to learn and network at SwiftLeeds, but understand VISA applications for the United Kingdom can sometimes be challenging. We will always do everything we can to assist in your application, including providing an official invitation letter.</p>
+              <p>To request this, please send an email to us at <a href="mailto:info@swiftleeds.co.uk?subject=VISA Invitation">info@swiftleeds.co.uk</a> with the subject 'VISA Invitation'.</p>
+              <hr />
+              
+              <!-- Refund Ticket -->
+              <h2 id="refund" class="h5 text-primary pt-1 pt-lg-3 mt-4">Ticket refunds</h2>
+              <p>SwiftLeeds will always refund 100% of ticket purchases (minus our 3% processing fee) if requested more than 14&nbsp;days before the conference begins. This year, that deadline is #(refund.deadline).</p>
```

Comment:
Review Copy

Reply:
Let's make this 30 days before the conference 👍🏼 which is about the time we have all the merch etc..